compiler/src/iree/compiler/Codegen/LLVMCPU/test/select_x86_64_lowering_strategy.mlir

```diff
@@ -1943,3 +1943,29 @@ module {
 //      CHECK:   iree_linalg_ext.custom_op
 // CHECK-SAME:       lowering_config = #[[CONFIG]]
 //  CHECK-NOT:   lowering_config
+
+// -----
+
+module {
```

Comment:
few nits:

- Remove the module op for consistency. (Other tests do not have module op.)
- Remove the `translation_info` and `lowering_config` from the input IR. Otherwise, we won't run the pass.
- I think it is generated by iree-compile? We can drop the annotation, i.e., rename the function name to `i1_type`.